test/pylib/start_s3_proxy.py

```diff
@@ -2,17 +2,21 @@
 import asyncio
 import signal
 import argparse
+
 from s3_proxy import S3ProxyServer
 
 
 async def run():
+    with open("/dev/urandom", "rb") as f:
+        seed = int.from_bytes(f.read(8), "big")
```

Comment:
could just
```py
import time
seed = int(time.time())
```
simpler this way.

Reply:
it has second resolution, meaning running tests in parallel may get the same seed. I want to avoid it

Reply:
ah, nm, actually the proxy starts only once, will switch to time() initialization